docker/Dockerfile

```diff
@@ -154,6 +154,9 @@ LABEL org.opencontainers.image.created=$BUILD_DATE
 ARG GITREF=unknown
 LABEL org.opencontainers.image.revision=$GITREF
 
+ARG USERID=10004
+ARG GROUPID=10004
```

Comment:
Sorry, this uid is taken (by ebmbot); 10005 should be OK

Reply:
Thanks, @rebkwok. I've updated to 10005.